collectors/collectors.go

```diff
@@ -37,6 +37,13 @@ var (
 		[]string{"resource"},
 	)
 
+	ObjectsListedTotalMetric = prometheus.NewCounter(
```

Comment:
As there are only a small amount of collectors it would make sense to add a label here to see which collector/resource type a specific counter comes from.